data/blog/bitcoin-infrastructure-this-decade.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: "Looking Ahead: Bitcoin Infrastructure This Decade"
+date: '2023-09-20'
+tags: ['OpenSats', 'bitcoin', 'infrastructure', 'research']
+draft: false
+authors: ['niftynei']
+images: ['/static/images/blog/09-nifty-infra.jpg']
+summary: "What will the next decade of bitcoin infrastructure building bring?"
```

Comment:
Perhaps better to use "Bitcoin's infrastructure" versus "bitcoin infrastructure?"